tests/utils/utils.test.jsx

```diff
@@ -1,18 +1,18 @@
 
 // Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
+//
+import {GeneralTypes} from 'mattermost-redux/action_types';
 
 import * as Utils from 'utils/utils.jsx';
+import store from 'stores/redux_store.jsx';
 
 describe('Utils.displayUsernameForUser', function() {
-    global.window.mm_config = {};
-
-    beforeEach(() => {
-        global.window.mm_config.TeammateNameDisplay = 'username';
-    });
-
     afterEach(() => {
-        global.window.mm_config = {};
+        store.dispatch({
+            type: GeneralTypes.CLIENT_CONFIG_RECEIVED,
+            data: {},
```

Comment:
Does this actually do anything?

Reply:
In this file, it ensures the state of the store doesn't impact subsequent tests. Ideally, everything would be run in isolation, but I haven't found a clean way to do this.

Reply:
Sorry, can you elaborate? I thought this action worked by overwriting the config with the properties present in `data` and leaving all others as-is (`Object.assign({}, state, action.data)`). It *looks* like a no-op to me, but if it isn't, can you maybe clarify the code path for me? What handles this action?

Reply:
Ah, you're right -- I didn't dig down deep enough into the redux reducer in question. I should be using `GeneralTypes.CLIENT_CONFIG_RESET` instead, though the fact that it doesn't matter suggests the tests don't care right now. I'll change it anyway to be future proof. 

Fixed -- thank you!